src/components/MainView/MainPage.jsx

```diff
@@ -127,13 +195,33 @@ export default function VerticalTabs() {
             value={value}
             onChange={handleChange}
           >
-            <StyledTab label="Networking" icon={<HubIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(0)} />
-            <StyledTab label="Security" icon={<SecurityIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(1)} />
-            <StyledTab label="Collaboration" icon={<GroupsIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(2)} />
-            <StyledTab label="Application Performance" icon={<SpeedIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(3)} />
+            <StyledTab onClick={() => handleTabClick("Networking")} label="Networking" icon={<HubIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(0)} />
+            <StyledTab onClick={() => handleTabClick("Security")} label="Security" icon={<SecurityIcon sx={{ fontSize: '2.5rem' }} />} {...a11yProps(1)} />
+            <StyledTab onClick={() => handleTabClick("Collaboration")} label="Collaboration" icon={<GroupsIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(2)} />
+            <StyledTab onClick={() => handleTabClick("Application Performance")} label="Application Performance" icon={<SpeedIcon sx={{ fontSize: '3rem' }} />} {...a11yProps(3)} />
           </StyledTabs>
         </Box>
-        <Box sx={{ flexBasis: '80%', marginTop: '4%', width: `${selectedView === 'Show sankey layout' ? '100%' : 'auto'}` }}>
+        <Box className="custom-box" sx={{ flexBasis: '80%', marginTop: '4%', width: `${selectedView === 'Show sankey layout' ? '100%' : 'auto'}` }}>
+          <TextField
+            style={{ width: 350, display: 'flex', marginLeft: '2.5rem', marginBottom: '1rem' }} color='black'
+            onChange={(e) => onInputChange(e.target.value)} className='search-field' id="standard-basic"
+            label="Search for a product..." variant="standard" value={searchValue}
+            sx={{
+              '& .MuiInput-underline:before': { borderBottomColor: '#cccbcb' },
+              '& .MuiInput-underline:after': { borderBottomColor: '#cccbcb' },
+              '& .MuiInput-underline:hover:before': { borderBottomColor: '#d4d2d2' },
+              '& .MuiInput-underline:hover:after': { borderBottomColor: '#d4d2d2' },
+            }}
+            slotProps={{
+              input: {
+                endAdornment: (
+                  <InputAdornment position="start">
+                    <Search sx={{ fill: '#cccbcb' }} />
+                  </InputAdornment>
+                ),
+              },
+            }}
+          />
```

Comment:
Same as above. Create a `styled-component`, such as for example `SearchTextField`, to address all CSS styling. Besides:
* remove `color: black` from the CSS styling
* I could not find `className=search-field` in any css file, is it still used? If not, please remove
* Do you still need a specific style to this component `<Search sx={{ fill: '#cccbcb' }} />`? Wouldn't this be inherited by its container? Please avoid adding too many styling changes in the code..

```suggestion
          <SearchTextField
            onChange={(e) => onInputChange(e.target.value)} 
            className='search-field'
            id="standard-basic"
            label="Search for a product..." variant="standard" value={searchValue}
            slotProps={{
              input: {
                endAdornment: (
                  <InputAdornment position="start">
                    <Search sx={{ fill: '#cccbcb' }} />
                  </InputAdornment>
                ),
              },
            }}
          />
```